flutter.el

```diff
@@ -129,13 +129,23 @@ ARGS is a space-delimited string of CLI flags passed to
    (let ((proc (get-buffer-process flutter-buffer-name)))
      (comint-send-string proc command))))
 
-(defun flutter--test (&optional args)
+(defun flutter--test (&rest args)
   "Execute `flutter test` inside Emacs.
 
-ARGS is a space-delimited string of CLI flags passed to
+ARGS is a list of CLI flags passed to
 `flutter`, and can be nil."
   (flutter--from-project-root
-   (compilation-start (format "%s test %s" (flutter-build-command) (or args "")) t)))
+   (compilation-start (format "%s test %s" (flutter-build-command) (mapconcat 'identity args " ")) t)))
+
+(defun flutter--find-test-case (line)
+  "Search backwards for test name starting at LINE on current buffer."
+  (save-excursion
+    (goto-char (point-min))
+    (forward-line (1- line))
+    (end-of-line)
+    (if (re-search-backward (concat "^[ \t]*\\(testWidgets\\|test\\|group\\)"
```

Comment:
Is there a reason you go to the end of the line and search backwards, rather than searching forwards from the beginning of the line?

Reply:
Hum, I think the both ways work, right?
I checked it on the `ruby-test-mode` and it's searching backward.

Reply:
It just seemed less intuitive to me, but there's probably a reason.

(I'm OK with "that's what `ruby-test-mode` does being *our* reason at the moment.)